torch/_export/non_strict_utils.py

```diff
@@ -80,6 +80,16 @@ def fakify(
     return fake
 
 
+def make_fake_params_buffers(
+    fake_mode: FakeTensorMode,
+    params_buffers: Dict[str, torch.Tensor],
+) -> Dict[str, Union[torch.Tensor, torch.nn.Parameter]]:
+    faked_params_buffers = {}
+    for key, value in params_buffers.items():
+        faked_params_buffers[key] = fake_mode.from_tensor(value, static_shapes=True)
```

Comment:
Should we be using the fake mode from the already fakified params and buffers instead of using the freshly created fake mode? I am not sure if it matters in practice tho.